### DISCUSSION TITLE
Feature request: allow editing via the GUI

### DISCUSSION THREAD

**Discussion by unikitty37** [elchininet/custom-sidebar#260] [Ideas]
I posted in the HA forum about my frustration with HA resetting the sidebar on a whim, and had custom-sidebar recommended to me.

However, it appears I have to write YAML or JSON to configure it, rather than dragging things into the right order and clicking Save, like in HA. That's considerably more of a pain :)

Do you have any plans to add GUI-based editing?

**Comment by elchininet** [comment]
>However, it appears I have to write YAML or JSON to configure it, rather than dragging things into the right order and clicking Save, like in HA. That's considerably more of a pain :)
>
>Do you have any plans to add GUI-based editing?

Hi @unikitty37,
No plans to do that. This is not an integration and it is not a custom-card either. As there is no support to build a UI for the sidebar from Home Assistant, that would require to build it completely from scratch, and that would mean that this project would become basically that UI functionality and the current functionality would be just a tiny layer on top of that. 🙂
